Makefile.js

```diff
@@ -46,25 +46,25 @@ target.lint = function() {
         lastReturn;
 
     echo("Validating Makefile.js");
-    lastReturn = nodeCLI.exec("eslint", MAKEFILE);
+    lastReturn = nodeCLI.exec("eslint", MAKEFILE, process.argv.slice(3));
```

Comment:
Assigning `process.argv.slice(3)` to a variable that is reused (`cliFlags`, maybe?) will make it less error prone in the future (since we'll only have to change it in one place). Do you mind updating that? Otherwise, this LGTM!